spec/javascripts/set-ga-client-id-on-form.spec.js

```diff
@@ -0,0 +1,23 @@
+/* global describe beforeEach it spyOn expect */
```

Comment:
Is `spyOn` needed here?

Reply:
No, I grabbed this from another spec file. Should I remove the ones I didn't use?

Reply:
Yeah, just spyOn, everything else is used.